util/packer/ubuntu-14.04/scripts/upgrade.sh

```diff
@@ -15,11 +15,6 @@ if [[ "${PACKER_BUILDER_TYPE}" == "virtualbox-ovf" ]]; then
   apt-get remove --purge -y virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
 fi
 
-apt-get install --install-recommends linux-generic-lts-utopic \
-  -y \
-  -o Dpkg::Options::="--force-confdef" \
-  -o Dpkg::Options::="--force-confold"
-
```

Comment:
I'm not sure I follow? The `dist-upgrade` below can still result in a new kernel.